gap-analysis/index.md

```diff
@@ -369,51 +369,51 @@ Such an attribute could be used by an authoring tool, for example, to encode the
+    <msup arg="x" intent="primed($1)">
       <mi arg="1">A</mi>
       <mo>&#x2032;</mo>
     </msup>
     <mo>&#x2063;</mo>
-    <msup arg="y" intent="prime($1)">
+    <msup arg="y" intent="primed($1)">
       <mi arg="1">B</mi>
       <mo>&#x2032;</mo>
     </msup>
   </mrow>
   <mo>¯</mo>
 </mover>
 ```
-
-where the name “segment” is used here to mean line segment, and the name “prime” is used to denote a primed variable symbol.
-
-Other meanings for the overbar notation would replace “segment” with another operator in the intent attribute.
+</details>
 
 
-The x-coordinate of the point B' could be encoded as:
+<details markdown="1">
+<summary><span markdown="1">X-coordinate example $B'_x$ </span></summary>
+This is similar to the line segment example in terms of complexity. Note that the second argument is meant to be spoken first.
 
 ```xml
-<msub intent="index($p,$v)">
-  <msup arg="p" intent="prime($1)">
+<msub intent="coordinate-of($p,$v)">
+  <msup arg="p" intent="primed($1)">
     <mi arg="1">B</mi>
     <mo>&#x2032;</mo>
   </msup>
   <mi arg="v">x<mi>
 </msub>
 ```
-
-where the name “index” is used here to select a component from a tuple, in this case, the x-coordinate from a point.
-
-
+</details>
+<br/>
 Various proposals have been discussed for the syntax to be supported by the intent attribute, with varying trade-offs in markup convenience. One proposal is to include default intent values for existing MathML presentation forms and default names for math operators to reduce the amount of markup needed to specify the intent of an expression. For example $\frac{n!}{(n-1)!)} = n$ would not require the explicit use of `intent` but would be equivalent to a version that included `intent="factorial($1)"` on the `mrow`s in the fraction, etc. While the full encoding of the content markup for an expression is often complex, and often not included with the presentation, the inclusion of the intent markup for an expression is potentially much simpler to generate and easier to consume, as part of the presentation markup.
 
 
 An advantage of this proposal is that it can be implemented using current technology without changes to other web standards, other than to specify that the attribute should be part of the accessibility tree built by browsers. While the functional syntax [‘name(arg1, ...)’]  represents a MathML-specific encoding, it is relatively easy to generate and to consume.
 
+The `intent` attribute is meant to be passed to AT. This allows AT to decide what is appropriate based on the needs/preferences of the reader. It also avoids the problem of having to know whether to embed in `aria-label` speech geared to someone who can't see the screen (and hence needs begin/end cues) and someone who can see the screen and needs audio reinforcement to help them decode what they see. It also leaves it to AT to add prosody cues and forcing a long 'a' sound, something that can't currently can't be done with `aria-label`.
```

Comment:
This technique of using the `AT` abbreviation as a stand-in for the tools dealing with accessibility seems to be used a lot in the doc. It warrants a mention near the beginning that `AT` may be used for referring to the general concept of accessibility, as well as the tools tasked with creating accessible output, so that people don't wonder.

Reply:
> It also leaves it to AT to add prosody cues and forcing a long 'a' sound, something that can't currently can't be done with `aria-label`.

"leaves it to AT"? So it is the same situation as aria-label? Not getting the contrast here.

Unless you're considering to add the hints through the intent attribute, this may still be in the "unsolved problems" territory. The big helper is `<mi>` which tells you `a` is a variable I think. 

Btw, the same latin variable `a` will sound different if read in a Bulgarian document, in fact is is exactly the short determiner "a" sound you're used to hearing in "a sound".